spec/requests/api/v2/providers/courses_spec.rb

```diff
@@ -125,6 +128,9 @@
               "bursary_amount" => nil,
               "scholarship_amount" => nil,
               "about_accrediting_body" => nil,
+              "english" => provider.courses[0].english,
```

Comment:
I think it'd be better for these values to be explicitly specified...

Reply:
Yeah, I went back on fourth on this.